src/config/lightnet.mlh

```diff
@@ -0,0 +1,46 @@
+[%%define ledger_depth 35]
+[%%import "/src/config/curve/medium.mlh"]
+[%%import "/src/config/coinbase/realistic.mlh"]
+[%%import "/src/config/scan_state/point2tps.mlh"]
```

Comment:
Maybe reduce the scan state size as well to reduce memory requirements? You could use `src/config/scan_state/medium.mlh` if you think it is ok to have 8 transactions per block. Or maybe define a new one for 2^4?  `/src/config/scan_state/point2tps.mlh` is 2^7 transactions

Reply:
This will enable seeing snarked ledger transitions sooner, lower snark work requirements and may also help a lil with the reduced block time- less transactions to process